src/test/java/uk/gov/laa/pfla/StepDefinitions.java

```diff
@@ -3,41 +3,54 @@
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
-import org.apache.http.HttpResponse;
-import org.apache.http.client.config.RequestConfig;
-import org.apache.http.client.methods.HttpGet;
-import org.apache.http.impl.client.CloseableHttpClient;
-import org.apache.http.impl.client.HttpClientBuilder;
+import io.restassured.response.Response;
 import uk.gov.laa.pfla.utils.ServiceUtils;
 
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Can we remove wildcard imports as they will get picked up either by sonar cloud in future or google-java-formatter

Reply:
Done :)